renovate.json

```diff
@@ -1,37 +1,37 @@
 {
-  "extends": ["config:base", ":disableDependencyDashboard", ":preserveSemverRanges"],
-  "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": ["after 9am every weekday", "before 5pm every weekday"],
-  "enabledManagers": ["npm", "nvm", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
+  "reviewers": ["neilmendum", "timoj58", "natclamp-moj"],
   "packageRules": [
     {
       "matchManagers": ["npm"],
-      "matchPackageNames": ["@types/node"],
-      "matchUpdateTypes": ["major"],
-      "enabled": false
-    },
-    {
-      "matchDatasources": ["docker"],
-      "allowedVersions": "16.15-bullseye-slim"
+      "rangeStrategy": "bump"
     },
     {
-      "matchDatasources": ["docker-compose"],
-      "matchPackageNames": ["bitnami/redis"],
-      "allowedVersions": "5.0"
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major NPM dependencies",
+      "groupSlug": "all-npm-minor-patch",
+      "stabilityDays": 3
     },
     {
-      "matchDatasources": ["orb"],
-      "matchPackageNames": ["cimg/node"],
-      "allowedVersions": "16.15-browsers"
+      "matchPackageNames": ["typescript", "govuk-frontend"],
+      "rangeStrategy": "bump",
+      "stabilityDays": 0
     },
     {
-      "matchPackageNames": ["typescript"],
-      "rangeStrategy": "bump"
+      "matchManagers": ["npm"],
+      "matchPackageNames": ["@types/node"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
     },
     {
-      "matchPackageNames": ["govuk-frontend"],
-      "rangeStrategy": "bump"
+      "matchDatasources": ["docker-compose"],
+      "matchPackageNames": ["bitnami/redis"],
+      "allowedVersions": "5.0"
     }
-  ]
+  ],
+  "vulnerabilityAlerts": {
+    "stabilityDays": 0
```

Comment:
As per [Renovate docs](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts)

> Renovate can read GitHub's Vulnerability Alerts to customize its Pull Requests

So configure Renovate to raise PRs for these straight away